src/GeoQuery.ts

```diff
@@ -139,6 +140,9 @@ export class GeoQuery {
     opStr: GeoFirestoreTypes.WhereFilterOp,
     value: any
   ): GeoQuery {
+    // True if inequality filter is used
+    this._ineq = (opStr !== '==' && opStr !== 'array-contains') ? true : this._ineq;
```

Comment:
Can we yank this, I'd rather go straight back to using `return this._query.orderBy('g').startAt(query[0]).endAt(query[1]) as GeoFirestoreTypes.web.Query;`

So any inequality checks can be removed.

Reply:
Done. 

The argument was that by using `where()` instead of `startAt`, the "orderBy query modifiers" could be used by devs. Therefore I wanted to respect that change. However I agree and with the " inequality checks" the behaviour becomes inconsistent.